Add the `Online rankings history` articles (#4720)
ppy/osu-wiki: wiki/History_of_osu!/Online_Rankings/en.md

---
tags:
  - history
  - leaderboard
  - rank 1
  - rank #1
  - rankings
  - top player
---

# History of Online Rankings

> Initial versions of osu! was an mostly isolated experience, on [October 6, 2007](https://osu.ppy.sh/community/forums/topics/15?n=29), support for public online rankings was added for [osu!](/wiki/Game_mode/osu!). Since then, all [game modes](/wiki/Game_mode) has recieved [online rankings](https://osu.ppy.sh/rankings/osu/performance).

Reviewer: ```suggestion
Early versions of osu! were a mostly isolated experience, with no online leaderboards for anything until [October 6, 2007](https://osu.ppy.sh/community/forums/topics/15?n=29), when support for public online rankings was added for [osu!](/wiki/Game_mode/osu!). Since then, all [game modes](/wiki/Game_mode) have recieved [online rankings](https://osu.ppy.sh/rankings/osu/performance).
```
grammar was a bit wonky here

> Numerous players has held rank #1 in each game mode under different ranking systems. The current system used by all game modes for online rankings is [performance points (pp)](/wiki/Performance_points).

Reviewer: ```suggestion
Numerous players have held rank #1 in each game mode under different ranking systems. The current system used by all game modes for online rankings is [performance points (pp)](/wiki/Performance_points).
```

## ![][osu!] osu!

> *Full ranking history page: [osu!](osu!)*

Reviewer: ```suggestion
*Main page: [osu!](osu!)*
```
should use main page hatnote style, othwerwise format using for see

Author: Resolved with https://github.com/ppy/osu-wiki/pull/4720/commits/caccb677c88639e0e96ce9a84d8436f54ad442a9

> As of June 3 2021, on the [osu! performance rankings](https://osu.ppy.sh/rankings/osu/performance) the rank #1 player is ![][flag_AU] [rushia1](https://osu.ppy.sh/users/7562902/osu).

Reviewer: ```suggestion
As of June 3, 2021, on the [osu! performance rankings](https://osu.ppy.sh/rankings/osu/performance) the rank #1 player is ![][flag_AU] [rushia1](https://osu.ppy.sh/users/7562902/osu).
```
M/D/Y format requires commas between the day and year (goes for all other sentences like this)

Author: Resolved with https://github.com/ppy/osu-wiki/pull/4720/commits/caccb677c88639e0e96ce9a84d8436f54ad442a9
